stdlib/public/Concurrency/Task.swift

```diff
@@ -424,12 +415,50 @@ public func runAsync(_ asyncFun: @escaping () async -> ()) {
   runTask(childTask.0)
 }
 
-struct RawTaskFutureWaitResult {
-  let hadErrorResult: Bool
-  let storage: UnsafeRawPointer
-}
-
 @_silgen_name("swift_task_future_wait")
-func taskFutureWait(
+func _taskFutureWait(
   on task: Builtin.NativeObject
-) async -> RawTaskFutureWaitResult
+) async -> (hadErrorResult: Bool, storage: UnsafeRawPointer)
```

Comment:
Why that change? the RawTaskFutureWaitResult was just that:

```swift
struct RawTaskFutureWaitResult {
  let hadErrorResult: Bool
  let storage: UnsafeRawPointer
}
```

Reply:
Raw tuples are easier to deal with in the compiler because I was able to avoid looking up the name.

Reply:
I see, thanks 👍 I guess time to remove that struct as well then.